easybuild/easyconfigs/a/azure-cli/azure-cli-2.47.0-GCCcore-10.3.0-Python-3.9.5.eb

```diff
@@ -0,0 +1,158 @@
+# # 
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Arturo Gimeno <arturo.gimeno@hpcnow.com>, Carlos Alejandro Garcia, Erica Bianco
+# License::     GPL-v3.0
+# #
+
+easyblock = 'PythonBundle'
+
+name = 'azure-cli'
+version = '2.47.0'
+
+homepage = 'https://learn.microsoft.com/en-us/cli/azure/'
+description = '''The Azure command-line interface (Azure CLI) is a set of commands used to create
+ and manage Azure resources. The Azure CLI is available across Azure services and is designed to
+ get you working quickly with Azure, with an emphasis on automation.'''
+
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
`versionsuffix` should be removed, we stopped using this to indicate Python 3 version in recent toolchains (we now only use it when Python 2.x is used as dependency)

Note that the easyconfig filename will need to be adjusted accordingly when this is removed